src/auth_service/auth_adapter/api/main.py

```diff
@@ -125,25 +125,25 @@ async def login(  # noqa: PLR0913
     else:
         access_token = get_bearer_token(authorization, x_authorization)
         try:
-            user_id, user_name, user_email = get_user_info(access_token)
+            ext_id, user_name, user_email = get_user_info(access_token)
```

Comment:
This also means the claim subject is now the external ID, right?

Reply:
Yes, the external id corresponds to the claim subject in the access token. It was like that before, but it was stored in the `user_id` field of the session. Now it is stored in a separate field.